cleanlab/regression/rank.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np 
+
+def get_label_quality_score(
+    true_labels: np.ndarray,
+    pred_labels: np.ndarray
+)-> np.ndarray:
+    """
+    Returns label quality score
+    
+    Score is continous value in range [0,1]
+    
+    1 - clean label (given label is likely correct).
+    0 - dirty label (given label is likely incorrect).
+    """
```

Comment:
raise ValueError if the two arrays are not same length?

Reply:
added in commit: [0503595](https://github.com/krmayankb/cleanlab/pull/1/commits/05035950d782fee4adde8da6c44afb47953fb5db)